config/crd/bases/infrastructure.cluster.x-k8s.io_metal3clusters.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.4
+    controller-gen.kubebuilder.io/version: v0.12.1
```

Comment:
this as actually one patch version ahead of CAPI 1.5 branch but I guess this is fine.